Localize policy as code #1653 (#2265)
cncf/glossary: content/it/policy-as-code.md

---
title: Policy as Code (PaC)
status: Completed
category: Concetto
tags: ["metodologia", "", ""]
draft: 
---

## Cos'è

Policy as code è la pratica che consente di memorizzare la definizione delle policy in uno o più file in formato leggibile e processabile da macchine. 
Questa metodologia sostituisce il modello tradizionale in cui le policy sono documentate in formato leggibile da umani in documenti esterni al sistema.

## Quali problematiche affronta

> La costruzione di applicazioni e infrastrutture è spesso vincolata da molte policy che un'organizzazione definisce, 

Reviewer: ```suggestion
La costruzione di applicazioni e infrastrutture è spesso vincolata a molte policy che un'organizzazione definisce, 
```

ad esempio policy di sicurezza che vietano di memorizzare segreti nel codice sorgente, di eseguire un container con autorizzazioni di superutente 
o di memorizzare alcuni dati al di fuori di una specifica regione geografica. 

>  Per gli sviluppatori e i revisori è molto laborioso e soggetto a errori verificare manualmente le applicazioni e le infrastrutture rispetto alle policy documentate.

Reviewer: ```suggestion
Per gli sviluppatori e i revisori è molto laborioso e soggetto a errori verificare manualmente le applicazioni e le infrastrutture rispetto alle policy documentate.
```

I processi manuali non possono soddisfare i requisiti di reattività e scalabilità delle applicazioni native del cloud.

## In che modo aiuta

> Utilizzare Policy as Code la ripetibilità e riduce gli errori, a differenza di quanto avviene manualmente.

Reviewer: ```suggestion
Utilizzare Policy as Code consente la ripetibilità e riduce gli errori, a differenza di quanto avviene manualmente.
```

---
title: Policy as Code (PaC)
status: Completed
category: Concetto
tags: ["metodologia", "", ""]
draft: 
---

## Cos'è

Policy as code è la pratica che consente di memorizzare la definizione delle policy in uno o più file in formato leggibile e processabile da macchine. 
Questa metodologia sostituisce il modello tradizionale in cui le policy sono documentate in formato leggibile da umani in documenti esterni al sistema.

## Quali problematiche affronta

La costruzione di applicazioni e infrastrutture è spesso vincolata da molte policy che un'organizzazione definisce, 
ad esempio policy di sicurezza che vietano di memorizzare segreti nel codice sorgente, di eseguire un container con autorizzazioni di superutente 
o di memorizzare alcuni dati al di fuori di una specifica regione geografica. 

> Il processo di verifica manuale della loro appliaczione su applicazioni o infrastruttura da parte di sviluppatori e revisori 

Reviewer 2: ```suggestion
 Per gli sviluppatori e i revisori è molto laborioso e soggetto a errori verificare manualmente le applicazioni e le infrastrutture rispetto alle policy documentate.
```

> è complesso e soggetto a errori. 

Reviewer 2: ```suggestion

```

I processi manuali non possono soddisfare i requisiti di reattività e scalabilità delle applicazioni native del cloud.

## In che modo aiuta

> Utilizzare il policy as a code consente la ripetibilità ed automazione, riducendo gli errori rispetto alla loro applicazione manuale.

Reviewer 2: ```suggestion
Utilizzare Policy as Code la ripetibilità e riduce gli errori, a differenza di quanto avviene manualmente.
```

Reviewer 2: togliere quell'_a_ che precede Code
